brig/cmd/brig/commands/project_create.go

```diff
@@ -471,6 +472,21 @@ func projectAdvancedPrompts(p *brigade.Project) error {
 				Default: p.DefaultScriptName,
 			},
 		},
+		{
+			Name: "brigadejsPath",
+			Prompt: &survey.Input{
+				Message: "Brigade.js relative path",
```

Comment:
I would word this as "brigade.js file path relative to the repository root".

Reply:
fixed, thanks!